File: JavaScript/8js 2/gvoc.js
/*
|--------------------------------------------------------------------------
| Volume of a Cuboid
|--------------------------------------------------------------------------
|
*/

function getVolumeOfCuboid(length, width, height) {
   return length * width * height;
}


const Kata = {
   getVolumeOfCuboid: (length, width, height) => length * width * height
}


const Kata = (() => {
   this.getVolumeOfCuboid = (l, w, h) => l * w * h;
   return this;
})();


const Kata = {
   getVolumeOfCuboid(length, width, height) {
      return length * width * height
   }
}


console.log(getVolumeOfCuboid(1, 2, 2), 4);
console.log(getVolumeOfCuboid(6.3, 2, 5), 63);

File: Python/8py 2/gvoc.py
"""
|--------------------------------------------------------------------------
| Volume of a Cuboid
|--------------------------------------------------------------------------
|
"""

def getVolumeOfCubiod(length, width, height):
   return length * width * height


getVolumeOfCubiod = lambda l,w,d : l*w*d


from functools import reduce
getVolumeOfCubiod = lambda l,w,h: reduce(lambda x,y:x*y, [l,w,h])


getVolumeOfCubiod = lambda *d: ( (lambda ft, op: ft.reduce(op.mul, d))  (__import__('functools'), __import__('operator')))


import math
def getVolumeOfCubiod(*args):
   return math.prod(args)


print(getVolumeOfCubiod(1, 2, 2), 4)
print(getVolumeOfCubiod(6.3, 2, 5), 63)